redisson/src/test/java/org/redisson/RedissonStreamTest.java

```diff
@@ -348,6 +348,42 @@ public void testRemoveMessages() {
         assertThat(stream.size()).isZero();
     }
 
+    @Test
+    public void testRemoveMessages2() {
+        GenericContainer<?> redis = createRedisWithVersion("redis:8.2.0");
```

Comment:
Do we need that? By default redis:latest is used. I think your current redis latest image isn't up to date. Can you check that?